CONTRIBUTING.md

```diff
@@ -33,10 +33,33 @@ Thank you for your interest in contributing to the CUDA C++ Core Libraries (CCCL
    For more details on building and testing, refer to the [Building and Testing](#building-and-testing) section below.
 
 4. **Commit Changes**:
+
+   Setup ssh signing keys (if needed, see next section) and commit:
+
    ```bash
    git commit -m "Brief description of the change"
    ```
 
+### SSH Signing Keys
+
+CCCL projects require commits to be signed with a cryptographic key.
```

Comment:
comment: This isn't quite true. Signed commits are only necessary for internal contributors who want CI to run automatically and not require `/ok to test`.

Reply:
Ah, ok, I'd misunderstood that nuance. I'll gladly move it to the ci docs, as long as I have something concise to link (and refer back to myself 😄)